src/app/forms/templates/general/notes.template.ts

```diff
@@ -1,15 +1,39 @@
-import { FormNode, FormNodeTypes, FormNodeViewTypes } from '../../services/dynamic-form.types';
+import { ValidatorNames } from '@forms/models/validators.enum';
+import { FormNode, FormNodeEditTypes, FormNodeTypes, FormNodeViewTypes } from '../../services/dynamic-form.types';
 
-export const NotesTemplate: FormNode = {
-  name: 'notesSection',
-  label: 'Notes',
-  type: FormNodeTypes.GROUP,
-  viewType: FormNodeViewTypes.SUBHEADING,
-  children: [
+export function getNotesTemplate(isPsv: boolean = false): FormNode {
+  let form: FormNode = {
+    name: 'notesSection',
+    label: 'Notes',
+    type: FormNodeTypes.GROUP,
+    viewType: FormNodeViewTypes.SUBHEADING,
+    children: []
+  }
+
+  if(isPsv) {
+    form.children?.push({
+      name: 'remarks',
+      label: 'Notes',
+      type: FormNodeTypes.CONTROL,
+      viewType: FormNodeViewTypes.STRING,
+      editType: FormNodeEditTypes.TEXTAREA,
+      validators: [{ name: ValidatorNames.MaxLength, args: 800 }]
+    },
     {
-      name: 'notes',
+      name: 'dispensations',
       type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.FULLWIDTH
-    }
-  ]
+      label: 'Dispensations',
+      viewType: FormNodeViewTypes.STRING,
+      editType: FormNodeEditTypes.TEXTAREA,
+      validators: [{ name: ValidatorNames.MaxLength, args: 160 }]
+    })
+    return form;
+  }
+
+  form.children?.push({
+    name: 'notes',
+    type: FormNodeTypes.CONTROL,
+    viewType: FormNodeViewTypes.FULLWIDTH
+  })
+  return form;
```

Comment:
This can be simplified:
```suggestion
  const psvNotes: FormNode[] =  [{
    name: 'remarks',
    label: 'Notes',
    type: FormNodeTypes.CONTROL,
    viewType: FormNodeViewTypes.STRING,
    editType: FormNodeEditTypes.TEXTAREA,
    validators: [{ name: ValidatorNames.MaxLength, args: 800 }]
  },
  {
    name: 'dispensations',
    type: FormNodeTypes.CONTROL,
    label: 'Dispensations',
    viewType: FormNodeViewTypes.STRING,
    editType: FormNodeEditTypes.TEXTAREA,
    validators: [{ name: ValidatorNames.MaxLength, args: 160 }]
  }];

  const defaultNotes: FormNode[] = [{
    name: 'notes',
    type: FormNodeTypes.CONTROL,
    viewType: FormNodeViewTypes.FULLWIDTH
  }];

  return {
    name: 'notesSection',
    label: 'Notes',
    type: FormNodeTypes.GROUP,
    viewType: FormNodeViewTypes.SUBHEADING,
    children: isPsv ? psvNotes : defaultNotes
  };
```

Reply:
Just noticed that it's already been merged.